resources/moneropedia/node.md

```diff
@@ -11,13 +11,13 @@ summary: "a device on the Internet running the Monero software, with a full copy
 
 ### More Information
 
-Nodes participate in the Monero network and secure @transactions by enforcing the rules of the network. Nodes download the entire @blockchain to know what transactions have taken place. Nodes assist the network by relaying transactions to other nodes on the network. Nodes may also choose contribute to the Monero network by participating in crafting @blocks (this is called @mining). 
+Nodes participate in the Monero network and secure @transactions by enforcing the rules of the network. Nodes download the entire @blockchain to know what transactions have taken place. Nodes assist the network by relaying transactions to other nodes on the network. Nodes may also choose to contribute to the Monero network by participating in crafting @blocks (this is called @mining). 
 
-Mining is the process by which nodes create a block from the previously accepted block, transactions that are waiting to be processed in the @tx-pool, and the @coinbase. When a node believes it has crafted a valid block it will transmit the completed block to other nodes on the network and those nodes signal agreement by working on the next block in the chain. 
+Mining is the process by which nodes create a block from the previously accepted block, transactions that are waiting to be processed in the Transaction Pool, and the @coinbase-transaction. When a node believes it has crafted a valid block it will transmit the completed block to other nodes on the network and those nodes signal agreement by working on the next block in the chain. 
```

Comment:
s/Transaction Pool/transaction pool

`transaction pool` is not a proper noun.

Reply:
Fixed.